src/py42/_compat.py

```diff
@@ -16,7 +16,9 @@
     from urlparse import urljoin
     from urlparse import urlparse
 
+    original_str = str
     str = unicode
+    str_options = (original_str, str)
```

Comment:
Doing this because of the following flow:

1.) py42 module would import `str` from compat, which would really be unicode in python2.7
2.) Then, a client module would create a str (normally, not unicode) in their python2.7 script and they would pass that str into a query filter.
3.) That query filter would turn their perfectly normal str into the value `None` because it was not of an expected type (datetime, float, or unicode)... It was non intuitive of what was going wrong.

Now, the new flow is for our pythong module to import `str_options` in this case for using `isinstance()` to verify that argument is one of the two str types.